babel.config.js

```diff
@@ -0,0 +1,71 @@
+module.exports = function(api) {
```

Comment:
Running the suggested `bundle exec rails webpacker:install` installed a lot of these new files.